### ISSUE TITLE
broken font at Splash Screen (Mac)

### ISSUE THREAD

**Issue by Alexander-Dus** [mediathekview/MediathekView#499]
![MT Splash](https://user-images.githubusercontent.com/62999956/78141313-9f1b3e80-742b-11ea-8b6e-cb9fffd2e65c.jpg)

The Splash Screen on Mac OS 10.12.6 seems broken (updated from 13.3.0 to 13.5.1), new font?

**Comment by Observerist** [comment]
Mine is looking "different". 

![Screen Shot 2020-01-22 at 00 57 19](https://user-images.githubusercontent.com/45834366/78306990-11cc0d00-7545-11ea-9d18-24b66eb210a9.jpeg)

Just glad I kept 11.555, every time I'm back on this version, I feel liberated from stress. 
Sorry for not loving 13.x.x. Feel bad 'bout that, The idea of this app is still great.

We Mac users seem to be only a handful here. Sad.

**Comment by styroll** [comment]
>@Observerist said: We Mac users seem to be only a handful here. Sad.

The developer of MV is a Mac user. The current version runs great here on macOS 10.14.6 displaying the splash screen correctly.

At least you have to submit some info what your OS and Java is concerned.

**Comment by Alexander-Dus** [comment]
If it helps: I mentioned OS 10.12.6 above, the Java version seems to be 15.0.1
Howver, I did not runinto any functional issue, it seems to be more cosmetic.
Mentioned it originally in case it's easy to fix :-)

**Comment by Alexander-Dus** [comment]
sorry, closing was by accident...

**Comment by derreisende77** [comment]
MV is developed primarily on macOS Now Catalina. So far there have been 3 reports of broken fonts for macOS and none can be reproduced on my MBP 16” or my 2011 MBP 15. Also various iMacs do not reproduce this problem. The next release of MV will use the final version of JavaFX 14 which might fix this inconvenience. For now I am going to close this issue as I cannot fix it with a patch

**Comment by styroll** [comment]
> If it helps: I mentioned OS 10.12.6 above

As you see in my post you are not the one who was addressed…

**Comment by Observerist** [comment]
Very refreshing to know that the Dev is on macOS. 

I'm on 10.13.6 High Sierra, Java 1.8.0_241-07. 

I just found this Splash quite "ModernArty", I'm not bothered by it appearing like that, it's just a Splash, not an important Window that went crazy.

I'm still using the old 11.555, the way the filters are used to make preselections pleases me more.
That doesn't mean, as I stated before, that the revised Mediathekview isn't a great app, it just went too far for me on certain functionalities.

Again, this is not a catastrophe, the real catastrophe is a very real virus threatening all of us.

I just had time to visit this forum, away from my usual workload. 
Just let us make the best of life and stay safe. Not bothering for some little detail.

Take care y'all !!!
